src/components/AthleteProfile/AthleteProfile.js

```diff
@@ -1,30 +1,23 @@
-import React from 'react';
 import './AthleteProfile.css';
 import Header from '../Header/Header';
-import { Route, Switch, useParams, useRouteMatch } from 'react-router';
-import EditPlayerProfile from './EditPlayerProfile/EditPlayerProfile';
-import { Link } from 'react-router-dom';
+import CarouselComp from '../Carousel/Carousel';
+import AthleteStatContainer from '../AthleteStatContainer/AthleteStatContainer';
+import Bio from '../Bio/Bio';
 
-const AthleteProfile = ( ) => {
+const AthleteProfile = ({ data }) => {
+  
 
-  const {id} = useParams()
-
-  const { path, url } = useRouteMatch()
-  console.log('path: ', path)
-  console.log('url: ', `${url}/editprofile`)
-  // console.log(useRef())
   return (
     <>
       <Header />
-      <h1>Athlete PROFILE id: {id}</h1>
-      <ul>
-        <li>
-          <Link to={`${url}/editprofile`} > Edit Profile</Link>
-        </li>
-      </ul>
-      <Switch>
-        <Route exact path={`${path}/editprofile`} component={EditPlayerProfile}/>
-      </Switch>
+      <main>
+        <AthleteStatContainer athlete={data} />
+        <div className='divider'></div>
+        <CarouselComp data={data}/>
+        <div className='divider'></div>
+        <Bio data={data} />
```

Comment:
This tag is very interesting, i have never use it before, I really want to take a look to it !